tests/alerts/test_ssh_password_auth_violation.py

```diff
@@ -43,16 +43,6 @@ class TestAlertSSHPasswordAuthViolation(AlertTestSuite):
         )
     )
 
-    events = AlertTestSuite.create_events(default_event, 10)
```

Comment:
We need to modify alerts/ssh_password_auth_violation.py to remove _type as well.

Reply:
This particular alert never fires, it will be part of our discussion on Wed.

Reply:
Per our discussion, this  is ok if it never fires (this is good) and I have modified _type to be category instead.